docs/source/multi_gpu.rst

```diff
@@ -341,7 +341,7 @@ project module). However, we STRONGLY discourage this use because it has limitat
           packages=find_packages()
           )
 
-Then setup your project like so:
+2. Setup your project like so:
 
 .. code-block:: bash
```

Comment:
btw, to make it look nice in the rendered output, you can indent the code block to the same level as the enumeration. I usually put a tab space after the number, like so:
1. (tab)  text text text
(tab)     here goes  code  block

Reply:
thanks!